### ISSUE TITLE
Alexa not discovering

### ISSUE THREAD

**Issue by CurtisFeatures** [thehookup/Motorized_MQTT_Blinds#10]
Hi, great work here as usual. I wonder if you could help, I have been using the MQTT blinds and it’s been working great. I’m trying to try out the Alexa version but having issues. I can flash the board and connect it to my WiFi giving it a name and everything. I can ping it so I know it’s connected to the WiFi however I’m unable to discover it in the Alexa app. I thought it might have been my WiFi at work causing issues but I’ve brought it home and reflagged and get the same issue here. I’ve never used Alexa before (google fan boy) so I’m not sure if I’m missing something.

**Comment by CurtisFeatures** [comment]
Iv made a bit of progress with this. I thought I would try flashing the fauxmoESP code onto the ESP8226 to see if I could get that to work. I didn't and had the same issues. Then I came across this issue post on BitBucket https://bitbucket.org/xoseperez/fauxmoesp/issues/58/v30-not-discovered-by-alexa with people having the same issue. Once I rolled back my board manager to 2.3.0 and changed by board rate to flash I was able to discover my device. 

Now when I try to compile the hook ups code with the 2.3.0 board manager I'm getting a lot of error to do with WiFimanger and Webserver but I think this might be to do with mismatched versions so im going to try roll some of those back and see if it works.
